more/free-programming-cheatsheets.md

```diff
@@ -327,6 +327,7 @@
 
 * [All in one Cheatsheet of Django](https://cheatography.com/ogr/cheat-sheets/django/pdf/) - Olivier R. (PDF)
 * [Django Quick Reference](https://www.codewithharry.com/blogpost/django-cheatsheet/) - Code with Harry (HTML)
+* [Django Quick Reference Guide](https://codemy.com/books/django-quick-reference-guide.pdf) - Codemy.com (PDF)
```

Comment:
This looks more like a brochure for the paid courses, than a cheat sheet. Would it be a value add?

Reply:
The first pages of the book contain valuable quick reference to django.

Reply:
delete it

Reply:
@eshellman I've deleted the quick reference link